docs/1000.yaml

```diff
@@ -4036,6 +4039,10 @@ Q107709489:
 
 Q107710112:
   title: Cantor's isomorphism theorem
+  decl: Order.iso_of_countable_dense
```

Comment:
I checked: this result matches wikipedia - but the declaration doc-string does not mention the theorem. Can you add it there as well?